### DISCUSSION TITLE
JSON.parse: bad control character in string literal at line 8 column 28 of the JSON data

### DISCUSSION THREAD

**Discussion by Dave0616** [rern/rAudio#1335] [Bug Reports]
I setup wifi and now I get this when I go to the network page

20230603

this is on firefox, 

Errors: JSON.parse: bad control character in string literal at line 8 column 28 of the JSON data 

{ 
  "page"        : "networXks"
, "activebt"    : 1
, "activeeth"   : true
, "activewl"    : true
, "camilladsp"  : false
, "connectedwl" : true
, "gateway"     : "10.0.0.1
10.0.0.1"
, "ipeth"       : "10.0.0.27"
, "ipsub"       : "10.0.0."
, "ipwl"        : "10.0.0.156"
, "listbt"      : false
, "listeth"     : {
  "gateway"  : "10.0.0.1
10.0.0.1"
, "hostname" : "rAudio.local"
, "ip"       : "10.0.0.27"
, "static"   : false
}
, "listwl"      : [ {
	  "dbm"      : -36
	, "gateway"  : "10.0.0.1"
	, "ip"       : "10.0.0.156"
	, "ssid"     : "removed"
	} ]
, "hostapd"     : false
, "hostname"    : "rAudio"
, "wldev"       : "wlan0"
, "login" : false }

this is in chrome

Errors: Bad control character in string literal in JSON at position 172

{ 
  "page"        : "networks"
, "activebt"    : 1
, "activeeth"   : true
, "activewl"    : true
, "camilladsp"  : false
, "connectedwl" : true
, "gateway"     : "10.0.0.1X
10.0.0.1"
, "ipeth"       : "10.0.0.27"
, "ipsub"       : "10.0.0."
, "ipwl"        : "10.0.0.156"
, "listbt"      : false
, "listeth"     : {
  "gateway"  : "10.0.0.1
10.0.0.1"
, "hostname" : "rAudio.local"
, "ip"       : "10.0.0.27"
, "static"   : false
}
, "listwl"      : [ {
	  "dbm"      : -36
	, "gateway"  : "10.0.0.1"
	, "ip"       : "10.0.0.156"
	, "ssid"     : "removed"
	} ]
, "hostapd"     : false
, "hostname"    : "rAudio"
, "wldev"       : "wlan0"
, "login" : false }

**Comment by Dave0616** [comment]
I just unplugged my wired connection and it seems okay, this seems to only happen when both wired and wireless connections are one ( im just now trying this and was still setting things up)

**Reply by rern** [reply]
It's a bug.
To be fixed by next update.

**Reply by Dave0616** [reply]
okay thank you!
